robolectric-resources/src/main/java/org/robolectric/manifest/MetaData.java

```diff
@@ -34,7 +32,7 @@ public MetaData(List<Node> nodes) {
     }
   }
 
-  public void init(ResourceTable resourceTable, String packageName) {
+  public void init(ResourceTable resourceTable, String packageName) throws RoboNotFoundException {
```

Comment:
We could just copy these values as plain strings and dereference then later in PackageManager